Supplier/OpenTRANS/OpenTRANS2_1_SHORT/Opentrans_v2.1_short_OrderResponse.adoc

```diff
@@ -0,0 +1,185 @@
+|===================================================================================
+
+[[OrderResponseItem]]
+=== Order Response Item
+
+If the line quantity is set to zero or less, the line is considered canceled.
+
+.Order Item Elements
+[width="100%",options="header",cols="3,2,2,1"]
+|=======================================================================
+| Element name             | Description            | Example             | Mandatory
+| LINE_ITEM_ID             | Line number from Order | 10                  |
+| PRODUCT_ID               | Product IDs            | <<ProductID>>       | Yes
+| QUANTITY                 | Amount ordered         | 5                   | Yes
+| ORDER_UNIT               | Unit, always p. piece  | C62 (fix)           |
+| PRODUCT_PRICE_FIX        | Price without VAT      | <<ProductPriceFix>> |
+| PRICE_LINE_AMOUNT        | total of line, without VAT | 88              |
+| DELIVERY_DATE            | delivery dates         | <<DeliveryDate>>    |
+|=======================================================================
+
+NOTE: If quantity is smaller or equals to 0, then the item is marked as cancelled
+
+NOTE: If DELIVERY_END_DATE is know, status is PROCESSING, otherwise it is UNKNOWN
+
+
+<<<
+[[ProductID]]
+==== Product ID
+
+.Product Id elements
+[width="90%",options="header"]
+|=======================================================================
+| Element name      | Description               | Example        | Remarks
+| SUPPLIER_PID      | Product id by supplier    | abc1234        | required when no LINE_ITEM_ID is provided
+| BUYER_PID         | Product id by buyer       | abc1234        | Expected <<GlossSku, sku>>
+| INTERNATIONAL_PID | EAN Code, only if present | 7109806181210  | ean
+|=======================================================================
+
+.Example:
+[source,xml]
+----
+<PRODUCT_ID>
+    <ns2:SUPPLIER_PID type="supplier_specific">AA123456</ns2:SUPPLIER_PID>
+    <ns2:INTERNATIONAL_PID type="ean">7109806181210</ns2:INTERNATIONAL_PID>
+    <ns2:BUYER_PID type="buyer_specific">123456</ns2:BUYER_PID>
+</PRODUCT_ID>
+----
+
+[[ProductPriceFix]]
+==== Product Price fix
+Product prices are always without vat.
+
+.Product price fix elemet
+[width="90%",options="header"]
+|=======================================================
+| Element name  | Description     | Example
+| PRICE_AMOUNT  | Price per unit  | 80.85
+|=======================================================
+
+.Example:
+[source,xml]
+----
+<PRODUCT_PRICE_FIX>
+    <ns2:PRICE_AMOUNT>80.85</ns2:PRICE_AMOUNT>
+</PRODUCT_PRICE_FIX>
+----
+
+<<<
+[[DeliveryDate]]
+==== Delivery Date
+The delivery end date will be considered as delivery date.
+
+.Delivery Date elements
+[width="90%",options="header"]
+|=======================================================
+| Element name         | Description              | Example
+| DELIVERY_START_DATE  | requested delivery date  | 2021-10-27T00:00:00
```

Comment:
these examples don't match the sample in the appendix (`2021-10-27` vs `2021-10-27`)